gatsby-node.js

```diff
@@ -41,6 +41,22 @@ exports.onCreateNode = ({ node, getNode, actions }) => {
       name: "slug",
       value: slug
     })
+
+    let siteMetadata = getNode('Site').siteMetadata;
+    let srcLink = "";
+    if (gitRemoteNode) {
+      srcLink = gitRemoteNode.webLink;
+      srcLink = srcLink + "/blob/master/" + fileNode.relativePath;
+    } else {
+      srcLink = siteMetadata.srcLinkDefault;
+      srcLink = srcLink + "/blob/master/content/" + fileNode.relativePath;
+    }
```

Comment:
can this be `const` instead? https://eslint.org/docs/rules/prefer-const

```suggestion
    const srcBase = gitRemoteNode ? `${gitRemoteNode.webLink}/blob/master/` : `${siteMetadata.srcLinkDefault}/blob/master/content/`
```

and later on use it as

```jsx
    createNodeField({
      node,
      name: "srcLink",
      value: srcBase + fileNode.relativePath
    })
```